request/builder.js

```diff
@@ -623,6 +632,7 @@ class ContractExecutionRequestBuilder {
     request.setFunctionArgument(this.functionArgument);
     request.setUseFunctionIds(this.useFunctionIds);
     request.setFunctionIdsList(this.functionIds);
+    request.setNonce(this.nonce);
```

Comment:
Just a confirmation. Don't we need to validate the input nonce here?

Reply:
@jnmt 
Jun, Thank you for the reminder.
We should validate the nonce as well.
Sorry, I forgot this validation.
I fixed this in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/82/commits/2811cb815371bcd2e541c0b81848fba371410560

In this commit, there is a change in ClientService
https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/82/commits/2811cb815371bcd2e541c0b81848fba371410560#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L951

This is a temporary change for passing the current unit test.
The implementation (user can pass nonce to ClientService) will be updated in later's PR.

Thank you!